src/ckb-daemon/usb_mac.c

```diff
@@ -838,6 +931,8 @@ int usbmain(){
     if(iterator_hid)
         iterate_devices_hid(0, iterator_hid);
 
+    register_mouse_event_tap(mainloop);
```

Comment:
@tatokis made an important point that this is a preprocessor if clause, which means we would need to provide more than one standalone .pkg.